build/Helix/PrepareHelixPayload.ps1

```diff
@@ -0,0 +1,55 @@
+[CmdLetBinding()]
+Param(
+    [string]$Platform,
+    [string]$Configuration,
+    [string]$ArtifactName='drop'
+)
+
+$payloadDir = "HelixPayload\$Configuration\$Platform"
+
+$repoDirectory = Join-Path (Split-Path -Parent $script:MyInvocation.MyCommand.Path) "..\..\"
+$nugetPackagesDir = Join-Path (Split-Path -Parent $script:MyInvocation.MyCommand.Path) "packages"
+ 
+# Create the payload directory. Remove it if it already exists.
+If(test-path $payloadDir)
+{
+    Remove-Item $payloadDir -Recurse
+}
+New-Item -ItemType Directory -Force -Path $payloadDir
+
+# Copy files from nuget packages
+Copy-Item "$nugetPackagesDir\microsoft.windows.apps.test.1.0.181203002\lib\netcoreapp2.1\*.dll" $payloadDir
+Copy-Item "$nugetPackagesDir\taef.redist.wlk.10.51.200127004\build\Binaries\$Platform\*" $payloadDir
+Copy-Item "$nugetPackagesDir\taef.redist.wlk.10.51.200127004\build\Binaries\$Platform\CoreClr\*" $payloadDir
+New-Item -ItemType Directory -Force -Path "$payloadDir\.NETCoreApp2.1\"
+Copy-Item "$nugetPackagesDir\runtime.win-$Platform.microsoft.netcore.app.2.1.0\runtimes\win-$Platform\lib\netcoreapp2.1\*" "$payloadDir\.NETCoreApp2.1\"
+Copy-Item "$nugetPackagesDir\runtime.win-$Platform.microsoft.netcore.app.2.1.0\runtimes\win-$Platform\native\*" "$payloadDir\.NETCoreApp2.1\"
+Copy-Item "$nugetPackagesDir\MUXCustomBuildTasks.1.0.48\tools\$platform\WttLog.dll" $payloadDir
```

Comment:
This is probably not necessary anymore as it should be in the 10.57 version of TAEF (and we probably don't have to pull this package at all.... but I might leave it because I think it has PGO tasks in it...)